src/main/java/org/opensearch/performanceanalyzer/collectors/SearchBackPressureStatsCollector.java

```diff
@@ -0,0 +1,742 @@
+                    searchShardTaskStats
+                            .getResourceUsageTrackerStats()
+                            .get(HEAP_USAGE_TRACKER_FIELD_NAME)
+                            .getCancellationCount();
+            this.searchbp_shard_stats_resource_heap_usage_currentMax =
+                    searchShardTaskStats
+                            .getResourceUsageTrackerStats()
+                            .get(HEAP_USAGE_TRACKER_FIELD_NAME)
+                            .getCurrentMax();
+            this.searchbp_shard_stats_resource_heap_usage_rollingAvg =
+                    searchShardTaskStats
+                            .getResourceUsageTrackerStats()
+                            .get(HEAP_USAGE_TRACKER_FIELD_NAME)
+                            .getRollingAvg();
+            this.searchbp_shard_stats_resource_cpu_usage_cancellationCount =
+                    searchShardTaskStats
+                            .getResourceUsageTrackerStats()
+                            .get(CPU_USAGE_TRACKER_FIELD_NAME)
+                            .getCancellationCount();
+            this.searchbp_shard_stats_resource_cpu_usage_currentMax =
+                    searchShardTaskStats
+                            .getResourceUsageTrackerStats()
+                            .get(CPU_USAGE_TRACKER_FIELD_NAME)
+                            .getCurrentMax();
+            this.searchbp_shard_stats_resource_cpu_usage_currentAvg =
+                    searchShardTaskStats
+                            .getResourceUsageTrackerStats()
+                            .get(CPU_USAGE_TRACKER_FIELD_NAME)
+                            .getCurrentAvg();
+            this.searchbp_shard_stats_resource_elaspedtime_usage_cancellationCount =
+                    searchShardTaskStats
+                            .getResourceUsageTrackerStats()
+                            .get(ELAPSED_TIME_USAGE_TRACKER_FIELD_NAME)
+                            .getCancellationCount();
+            this.searchbp_shard_stats_resource_elaspedtime_usage_currentMax =
+                    searchShardTaskStats
+                            .getResourceUsageTrackerStats()
+                            .get(ELAPSED_TIME_USAGE_TRACKER_FIELD_NAME)
+                            .getCurrentMax();
+            this.searchbp_shard_stats_resource_elaspedtime_usage_currentAvg =
+                    searchShardTaskStats
+                            .getResourceUsageTrackerStats()
+                            .get(ELAPSED_TIME_USAGE_TRACKER_FIELD_NAME)
+                            .getCurrentAvg();
+        }
+
+        public void populate_task_stats(SearchTaskStats searchTaskStats) {
+            this.searchbp_task_stats_cancellationCount = searchTaskStats.getCancellationCount();
+            this.searchbp_task_stats_limitReachedCount = searchTaskStats.getLimitReachedCount();
+            this.searchbp_task_stats_resource_heap_usage_cancellationCount =
+                    searchTaskStats
```

Comment:
We can probably rewrite this part of code to make it more readable, for example, using some temp variables to replace 
```
searchTaskStats
                            .getResourceUsageTrackerStats()
                            .get(HEAP_USAGE_TRACKER_FIELD_NAME)

searchTaskStats
                            .getResourceUsageTrackerStats()
                            .get(CPU_USAGE_TRACKER_FIELD_NAME)

 searchTaskStats
                            .getResourceUsageTrackerStats()
                            .get(ELAPSED_TIME_USAGE_TRACKER_FIELD_NAME)

```

Reply:
```
searchTaskStats
                            .getResourceUsageTrackerStats()
                            .get(HEAP_USAGE_TRACKER_FIELD_NAME)
```
Has now been simplified to 
`task_heap_stats`

Similar simplification applies to other variables.